tests/test-helper.js

```diff
@@ -13,5 +13,4 @@ Object.keys(require.entries).forEach((entry) => {
 });
 
 setResolver(resolver);
-start();
-
+start();
```

Comment:
This needs a newline at the end of the file.

Reply:
Fixed